README.md

```diff
@@ -178,10 +178,13 @@ an error during startup, but will remain usable.
 
 For more information on the launch file's parameters see its own documentation.
 
-Once the robot driver is started, load the previously generated program on the robot panel and
-execute it. From that moment on the robot is fully functional. You can make use of the pause
-function or even stop the program. Simply press the play button again and the ROS driver will
-reconnect.
+Once the robot driver is started, load the previously generated program on the robot panel that will
+start the `External Control` programnode and execute it. From that moment on the robot is fully
+functional. You can make use of the pause function or even stop the program. Simply press the play
+button again and the ROS driver will reconnect.
+
+Inside the ROS terminal running the driver you should see the output `Robot ready to receive control commands.`
```

Comment:
```suggestion
Inside the ROS terminal running the driver you should see the output *Robot ready to receive control commands*.
```

Reply:
I decided against that, as we are referring to actual terminal output here.